src/app/components/AllowanceMenu/index.tsx

```diff
@@ -236,8 +239,8 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                   {t("edit_permissions")}
                 </h2>
                 <div>
-                  {permissions.map((permission) => (
-                    <>
+                  {permissions.map((permission, i) => (
+                    <Fragment key={`element-${i}`}>
```

Comment:
Can we use the permission id as the key here, too?

Reply:
It's not gonna appear anyways as it's a fragment, should I change it?

Reply:
Yeah, but since you already have a unique property `permission.id` I think there is no point in generating a new unique key based on the current index of the array. Please change if you agree or resolve the conversation if you don't.

Reply:
Ah, got it!